metaflow/plugins/kfp/kfp.py

```diff
@@ -278,8 +288,8 @@ def _step_cli(self, node, input_paths, user_code_retries, task_id):
             "--quiet",
             "--metadata=%s" % self.metadata.TYPE,
             "--environment=%s" % self.environment.TYPE,
-            "--datastore=%s" % self.datastore.TYPE,
-            "--datastore-root=%s" % self.datastore.datastore_root,
+            "--datastore=s3",
```

Comment:
Does this mean the datastore is defaulted to s3? If the user wants to change the datastore, where could they do it?

Reply:
KFP for now supports only supports s3 datastore.  This is how Metaflow state can be accessed between KFP step containers.